src/css/css-document.ts

```diff
@@ -88,11 +88,17 @@ export class ParsedCssDocument extends ParsedDocument<shady.Node, Visitor> {
     throw new Error('Not implemented');
   }
 
-  stringify() {
-    return cssbeautify(
-               shadyStringifier.stringify(this.ast),
-               {indent: '  ', autosemicolon: true, openbrace: 'end-of-line'}) +
+  stringify(options?: StringifyOptions) {
+    options = options || {};
+    const beautifulResults =
+        cssbeautify(
+            shadyStringifier.stringify(this.ast),
+            {indent: '  ', autosemicolon: true, openbrace: 'end-of-line'}) +
         '\n';
+
+    return beautifulResults.split('\n')
+        .map(line => line && '  '.repeat(options.indent || 0) + line)
```

Comment:
It's when it's the empty string, we don't want to splat out a bunch of trailing whitespace.